.github/workflows/algolia/src/index.ts

```diff
@@ -115,15 +78,36 @@ export const generateAlgoliaJson = () => {
   return jsonData;
 };
 
-export const uploadAlgoliaData = async (records) => {
+const uploadAlgoliaData = async (index: SearchIndex, records) => {
+  return await index.saveObjects(records).catch(console.error);
```

Comment:
You don't need `await` here since there is no more logic here to run after awaiting it. 

You would need to `return await` if this was in a `try ... catch` block, so that the `catch` block was triggered if `index.saveObjects` errored.

Reply:
I see, right. Since the function is returning a Promise which I'm awaiting at the callee.